modules/vba_documents/app/controllers/vba_documents/v2/uploads_controller.rb

```diff
@@ -96,7 +97,8 @@ def submit
 
           # Validations
           validate_parts(upload_model, parts)
-          validate_metadata(parts[META_PART_NAME], submission_version: upload_model.metadata['version'].to_i)
+          validate_metadata(parts[META_PART_NAME], upload_model.consumer_id, upload_model.guid,
```

Comment:
pass in the consumer_id to check to see if they have any allowed custom metadata, upload guid is for logging